docs/data-guidelines.md

```diff
@@ -256,3 +257,26 @@ If the table indicates an initial version of "1" and an information source says
 If you're adding new data for Node.js, use `0.10.0` or later. If a feature was added in a version before `0.10.0`, use `0.10.0` anyway.
 
 This guideline was proposed in [#6861](https://github.com/mdn/browser-compat-data/pull/6861).
+
+## Mixins
+
+Mixins appear in WebIDL and in specifications to define Web APIs. For web developers, they aren't observable directly; they act as helpers to avoid repeating API definitions. Don't add mixins to BCD where they do not already exist.
+
+For example, [`HTMLHyperlinkElementUtils`](https://html.spec.whatwg.org/multipage/links.html#htmlhyperlinkelementutils) is a mixin defined in the HTML specification.
+
+Members of this mixin are available to `HTMLAnchorElement` and `HTMLAreaElement`, that's where BCD exposes them. Add mixin members to BCD in one of these ways:
+
+1. For smaller mixins, add members of `HTMLHyperlinkElementUtils` directly to the `api/HTMLAnchorElement.json` and `api/HTMLAreaElement.json` files as if they were regular members of these interfaces.
+
+2. For larger mixins, create a file in the `api/_mixins/` folder and indicate for which interface they are using file names like: `HTMLHyperlinkElementUtils__HTMLAnchorElement.json` and `HTMLHyperlinkElementUtils__HTMLAreaElement.json`.  
+   In these files, expose the data under the correct tree. For `HTMLHyperlinkElementUtils__HTMLAnchorElement.json`, the file needs to start like this:
+
+```json
```

Comment:
The tiniest nit ever: I'm not sure why the diff paints this red. Maybe because the JSON is invalid? I looked at the rendered file on your branch directly and it seems fine, though that caused me to notice that this is not indented to the level of the `2.` body text. you could add three spaces to each of the code block lines to make it align to the parent list item.

Otherwise, marking this as approved, with the expectation that we'll merge it at the same time as the other PR. 👍

Reply:
I think it's red because the JSON isn't terminated. It's the same style as you get for invalid JSON, often if you put `...` in an example.

Reply:
I've fixed the indentation and I've removed the json highlighting, so that it doesn't try to process it.